server/log-collector/pkg/services/logcollector/server.go

```diff
@@ -712,7 +718,9 @@ func (s *Server) startLogStreaming(ctx context.Context,
 		if err != nil {
 			// if the pod is still running, it means the logs were rotated, so we need to get a new stream
 			// by bailing out
-			if !errors.Is(err, common.PodStillRunningError{}) {
+			if !errors.Is(err, common.PodStillRunningError{
```

Comment:
u dont need to construct the error, just pass it
`errors.Is(err, common.PodStillRunningError)`